web/src/edit/NeighbourhoodMode.svelte

```diff
@@ -248,36 +325,54 @@
       </mark>
     {/if}
 
-    <div style="display: flex; justify-content: space-between;">
+    <div style="display: flex; justify-content: space-between">
```

Comment:
I switched to icon buttons to make things fit, as discussed.

- bug 1: the tooltips get clipped by the map canvas. Not sure how to fix pico zordering without modifying its stylesheets, but probably a `:global(some class)` should do the trick
- bug 2: Zoom your browser in / out. The icons always stretch and get absurdly big or small. The responsive behavior is definitely kind of wrong. Getting the icon images to have the same size would probably be a good start.